apidef/api_definitions.go

```diff
@@ -73,6 +73,7 @@ const (
 	// For multi-type auth
 	AuthTypeNone  AuthTypeEnum = ""
 	AuthToken     AuthTypeEnum = "auth_token"
+	Custom        AuthTypeEnum = "custom"
```

Comment:
Why it is called custom, and not JSVM or similar?

Reply:
Can we enable coprocess auth with python, jsvm or goplugin at the same time? I think no. That's why we can combine them as `custom`. What do you think?

Reply:
Can we then call it custom_auth or custom_auth_plugin? 

But also it makes it work only for JSVM right, you will need to do the same logic for coprocess,grpc too?

Reply:
Done.